<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1485</width>
    <height>905</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="grammarEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="grammarCompleteBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>完成</string>
    </property>
   </widget>
   <widget class="QLabel" name="vtLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>vt集</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="firstvtLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>firstvt集</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lastvtLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>670</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>lastvt集</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="firstvtBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="lastvtBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="vtBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="relationLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>1171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>优先关系矩阵</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="relationBrowser">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>1171</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="codeEdit">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>471</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="codeLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入代码(空格和换行符、制表符等会自动被忽略)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="codeCompleteBtn">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>340</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>开始规约</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="parsingWidget">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>1161</width>
      <height>491</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>规约栈</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>剩余代码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>关系</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>动作</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>动作完成后的栈</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="vnLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>vn集</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="vnBrowser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>支持一行一文法以及一行多文法</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>即E-&gt;F和E-&gt;F|G都可以被正常解析</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>文法中的空格会被自动忽略</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>请输入文法</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>241</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>20</width>
      <height>881</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>1171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
